### ISSUE TITLE
[BUG]: Devtools panels randomly won't load

### ISSUE THREAD

**Issue by drjoho** [IBMa/equal-access#781]
### Project

a11y checker extension

### Browser

Chrome, Firefox

### Operating system

MacOS

### Description

The Checker to show up in the Chrome Dev Tools toolbar as shown

<img width="1866" alt="image" src="https://user-images.githubusercontent.com/64442976/160877949-533a4bbe-c571-4a1f-a984-60c6c066e3b5.png">

However, on other pages the Accessibility Checker tab is missing

<img width="1866" alt="image" src="https://user-images.githubusercontent.com/64442976/160878234-fb6fad08-863b-457c-b0fa-b2589e397fac.png">

Note that this is not just affecting our extension other third-party extensions are not showing either

### Steps to reproduce

Go to the site ibm.com/products
with the checker extension installed open devtools
you will see the following - note the "Accessibility Checker" is not there

<img width="1866" alt="image" src="https://user-images.githubusercontent.com/64442976/160878974-7c204400-2c98-44bf-955c-c75fb06007d6.png">

**Comment by tombrunet** [comment]
Opened with Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=1311690

**Comment by shunguoy** [comment]
Triage: Chrome issue, will need to watch the status of this.

**Comment by philljenkins** [comment]
The Checker extension also fails to load on other ibm.com pages, such as
https://www.ibm.com/us-en?ar=1

But, if I load another page that loads the Checker, such as https://demo.testfire.net/, and scan,
then load one of the following pages: https://www.ibm.com/us-en?ar=1, 
then since the Checker is already loaded, it will scan that page.

However, when loading the partner page, https://www.ibm.com/partners/
and the Checker is already loaded, the Scan button continues for ever and never completes.

**Comment by tombrunet** [comment]
@philljenkins The 'never completes' is a separate issue that is already fixed. This issues is about Chrome not loading the panel.

**Comment by tombrunet** [comment]
This will supposedly be fixed in Chrome 103 (the next version). Seems to be working in Chrome Canary.

**Comment by marcjohlic** [comment]
103 is out - @eirckr @tombrunet @marcjohlic will check different pages

**Comment by marcjohlic** [comment]
Closing - this seems to be corrected.
